src/com/company/Sudoku.java

```diff
@@ -0,0 +1,380 @@
+package com.company;
+
+import javax.swing.*;
+import java.util.Arrays;
+import java.util.Scanner;
+import java.util.regex.Pattern;
+
+public class Sudoku {
+    public Sudoku(Scanner gameScan) {
```

Comment:
już nie będę czepiał się enterów, po prostu klikaj ctrl alt l i dbaj o czytelność kodu, jak ja cię nie poprawię to będzie to robił ktoś inny. Enter między sekcjami, funkcjami to must have. Za dużo enterów znowu jest zbędne i czasami tylko warto rozdzielić dwoma enterami, ale raczej nie więcej.